app/assets/javascripts/channels/groups.js

```diff
@@ -0,0 +1,131 @@
+function initGroupCable() {
+  var messages = $('#group-messages');
+  if(messages.length <= 0) return;
+
+  var groupId = messages.data('group-id');
+  var userId = messages.data('user-id');
+  var template = JST['templates/message'];
+
+  var page = 2;
+  var totalPages = 37;
+  var loadingMessages = false;
+
+  var toBottom = function() {
+    messages.scrollTop(messages.prop("scrollHeight"));
+  };
+
+  var prependMessage = function(message) {
+    messages.prepend(template({message: message, current_user: userId}));
+  };
+
+  var loadMessages = function() {
+    $.getJSON('/grupper/' + groupId + '/meddelanden/')
+      .then(function(resp) {
+        for (var message of resp.messages) prependMessage(message);
+        totalPages = resp.meta.total_pages;
```

Comment:
Variables that come from the json data created in rails. Not going to edit that for some JS guidelines